tools/extras/install_cudnn.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# https://developer.nvidia.com/rdp/cudnn-download
+
+if [ ! -f srilm.tgz ]; then
```

Comment:
Also, I think we should use the original name (cudnn-7.0-linux-x64-v4.0-prod.tgz).
The reason is that we are going to use it as a library (I assume), so the API version will be important.